packages/rum-core/src/domain/view/trackViews.spec.ts

```diff
@@ -965,3 +948,24 @@ describe('view event count', () => {
     })
   })
 })
+
+function setupViewTracking(lifeCycle: LifeCycle, initialViewOptions: ViewOptions | undefined = undefined) {
```

Comment:
**💬 suggestion:** ‏`setupViewTest` already serves the same purpose of this function. Maybe merge the two functions?